.fixtures.yml

```diff
@@ -4,5 +4,6 @@ fixtures:
     "apt": "git://github.com/puppetlabs/puppetlabs-apt.git"
     "staging": "git://github.com/voxpupuli/puppet-staging.git"
     "erlang:": "git://github.com/garethr/garethr-erlang.git"
+    "rabbitmq_http_api_client": "https://github.com/ruby-amqp/rabbitmq_http_api_client"
```

Comment:
Interesting. I've never thought about adding a gem as a modules fixture. I think this should just be handled via the Gemfile though, and should not be put here.